### ISSUE TITLE
StrictMode violation in BraintreeFragment

### ISSUE THREAD

**Issue by neonixv** [braintree/braintree_android#161]
### General information

* SDK/Library version: 2.5.4

See stack trace

### Issue description

```
08-18 10:41:30.196 11612-11612/com.google.android.apps.gmm.dev D/StrictMode: StrictMode policy violation; ~duration=214 ms: android.os.StrictMode$StrictModeDiskReadViolation: policy=65567 violation=2
                                                                                 at android.os.StrictMode$AndroidBlockGuardPolicy.onReadFromDisk(StrictMode.java:1263)
                                                                                 at android.database.sqlite.SQLiteConnection.applyBlockGuardPolicy(SQLiteConnection.java:1039)
                                                                                 at android.database.sqlite.SQLiteConnection.executeForLong(SQLiteConnection.java:593)
                                                                                 at android.database.sqlite.SQLiteConnection.setJournalSizeLimit(SQLiteConnection.java:271)
                                                                                 at android.database.sqlite.SQLiteConnection.open(SQLiteConnection.java:214)
                                                                                 at android.database.sqlite.SQLiteConnection.open(SQLiteConnection.java:191)
                                                                                 at android.database.sqlite.SQLiteConnectionPool.openConnectionLocked(SQLiteConnectionPool.java:463)
                                                                                 at android.database.sqlite.SQLiteConnectionPool.open(SQLiteConnectionPool.java:185)
                                                                                 at android.database.sqlite.SQLiteConnectionPool.open(SQLiteConnectionPool.java:177)
                                                                                 at android.database.sqlite.SQLiteDatabase.openInner(SQLiteDatabase.java:806)
                                                                                 at android.database.sqlite.SQLiteDatabase.open(SQLiteDatabase.java:791)
                                                                                 at android.database.sqlite.SQLiteDatabase.openDatabase(SQLiteDatabase.java:694)
                                                                                 at android.app.ContextImpl.openOrCreateDatabase(ContextImpl.java:571)
                                                                                 at android.content.ContextWrapper.openOrCreateDatabase(ContextWrapper.java:269)
                                                                                 at android.database.sqlite.SQLiteOpenHelper.getDatabaseLocked(SQLiteOpenHelper.java:223)
                                                                                 at android.database.sqlite.SQLiteOpenHelper.getWritableDatabase(SQLiteOpenHelper.java:163)
                                                                                 at com.braintreepayments.api.internal.AnalyticsDatabase.addEvent(AnalyticsDatabase.java:61)
                                                                                 at com.braintreepayments.api.BraintreeFragment$1.onConfigurationFetched(BraintreeFragment.java:467)
                                                                                 at com.braintreepayments.api.BraintreeFragment$11.run(BraintreeFragment.java:681)
                                                                                 at com.braintreepayments.api.BraintreeFragment.postOrQueueCallback(BraintreeFragment.java:605)
                                                                                 at com.braintreepayments.api.BraintreeFragment.waitForConfiguration(BraintreeFragment.java:673)
                                                                                 at com.braintreepayments.api.BraintreeFragment.sendAnalyticsEvent(BraintreeFragment.java:463)
                                                                                 at com.braintreepayments.api.AndroidPay.tokenize(AndroidPay.java:194)
                                                                                 at com.braintreepayments.api.AndroidPay.tokenize(AndroidPay.java:179)
```

**Comment by crookedneighbor** [comment]
Thanks for reporting this, we're looking into it.

**Comment by sdcoffey** [comment]
This issue has been fixed as of release 2.7.1. Thanks for the report!
